src/App.tsx

```diff
@@ -8,11 +8,26 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
better to save tab's id here, because you save tab in 2 places. In tabs array and here

Reply:
This is condition of the task:
2 - Save the selectedTab in the App (the first one by default)
4 - Pass the selectedTabId as a prop to the Tabs, the specified tab should be selected if possible (otherwise the first tab is selected)